src/components/@shared/VerifiedBadge/index.tsx

```diff
@@ -22,11 +25,25 @@ export function Badge({
       className={cx({
         mainLabel: true,
         isValid,
+        isWarning: matchVerifiable === false,
         [className]: className
       })}
     >
+      {matchVerifiable === false && (
+        <Tooltip content="gx:dependsOn could not be found" />
+      )}
       <span>{verifiedService}</span>
-      {isValid ? <VerifiedPatch /> : <Cross />}
+      {typeof matchVerifiable === 'undefined' ? (
+        isValid ? (
+          <VerifiedPatch />
+        ) : (
+          <Cross />
+        )
+      ) : isValid && matchVerifiable ? (
+        <VerifiedPatch />
+      ) : (
+        <Cross />
+      )}
```

Comment:
suggestion(blocking): this could be simplified with a `false` check, e.g.
```tsx
{isValid && matchVerifiable !== false ? 
  //...
:
  // ... 
}
```